astore/server/auth/auth.go

```diff
@@ -38,6 +38,22 @@ type Server struct {
 	log                   logger.Logger
 }
 
+func (s *Server) HostCertificate(ctx context.Context, request *auth.HostCertificateRequest) (*auth.HostCertificateResponse, error) {
+	b, _ := pem.Decode(request.Hostcert)
+	pubKey, err := ssh.ParsePublicKey(b.Bytes)
```

Comment:
from the documentation of `pem.Decode`, `b` could be nil here? In that case, `b.Bytes` would probably crash?

I recommend adding a ceck `if b == nil { return .. ` some error.

Reply:
Ah nice catch, fixed